code/drasil-printers/Language/Drasil/Printing/Import.hs

```diff
@@ -136,11 +136,11 @@ expr (UnaryOp Cot u)    sm = mkCall sm P.Cot u
 expr (UnaryOp Arcsin u) sm = mkCall sm P.Arcsin u
 expr (UnaryOp Arccos u) sm = mkCall sm P.Arccos u
 expr (UnaryOp Arctan u) sm = mkCall sm P.Arctan u
-expr (UnaryOp Dim u)    sm = mkCall sm P.Dim u
-expr (UnaryOp Not u)    sm = P.Row [P.MO P.Not, expr u sm]
 expr (UnaryOp Exp u)    sm = P.Row [P.MO P.Exp, P.Sup $ expr u sm]
 expr (UnaryOp Abs u)    sm = P.Fenced P.Abs P.Abs $ expr u sm
-expr (UnaryOp Norm u)   sm = P.Fenced P.Norm P.Norm $ expr u sm
+expr (UnaryOpB Not u)    sm = P.Row [P.MO P.Not, expr u sm]
```

Comment:
When you edit a function, it is worth trying to keep the vertical alignment. It makes reading the code later much easier.

Reply:
I'll make sure to do this better in the future, thank you!